### PR TITLE
Testimonials

### PR DESCRIPTION
**Pull request by lizzietm**
Added a testimonials section for the landing page.  Which should be responsive for computer and mobile devices

### CODE REVIEW

File: conversationally/frontend/src/pages/landingPage/sections/Testimonials.jsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+const DemoPaper = styled(Paper)(({ theme }) => (
+    {
+        display: 'flex',
+        flexDirection: 'column',
+        alignItems: 'center',
+        justifyContent: 'flex-start',
+        width: '100%',
+        maxWidth: 300,
+        padding: theme.spacing(2),
+        margin: theme.spacing(1),
```
Comment (life-efficient) [inline conversationally/frontend/src/pages/landingPage/sections/Testimonials.jsx:18]: This seems unnecessary

**Review by life-efficient — CHANGES_REQUESTED** [review]

File: conversationally/frontend/src/pages/landingPage/sections/Testimonials.jsx (hunk 1 above)
Comment (lizzietm) [reply conversationally/frontend/src/pages/landingPage/sections/Testimonials.jsx:18]: Unecessary lines removed.

File: conversationally/frontend/src/pages/landingPage/LandingPage.jsx [hunk 2]
```diff
@@ -26,8 +27,32 @@ const Navbar = () => {
...
+        stars: 4,
+        name: "- Michael P",
+    },
+
+    {
+        text: "The future is now, Conversationally has been very helpful for my daily tasks!",
+        img: "",
+        stars: 5,
+        name: "-Morgan D",
+    },
```
[the remark this answers is not in the document]
Comment (lizzietm) [reply conversationally/frontend/src/pages/landingPage/LandingPage.jsx:52]: Will do and will make the changes.

File: conversationally/frontend/src/pages/landingPage/sections/Testimonials.jsx [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+                                {testimonial.text}
+                            </Typography>
+
+                            <Typography variant="h4" sx={{ margin: '10px ', typography: { sm: "h4", xs: "h6" } }} style={{ fontWeight: 700 }}>
+                                {testimonial.name}
+                            </Typography>
+
+                        </div>
+
+                    </DemoPaper>
```
[the remark this answers is not in the document]
Comment (lizzietm) [reply conversationally/frontend/src/pages/landingPage/sections/Testimonials.jsx:55]: Going to merge for the MVP and then work on this in the background.

File: conversationally/frontend/src/pages/landingPage/sections/Testimonials.jsx [hunk 4]
```diff
@@ -0,0 +1,63 @@
+import * as React from 'react';
+import Stack from '@mui/material/Stack';
+import Paper from '@mui/material/Paper';
+import Rating from '@mui/material/Rating';
+import Typography from '@mui/material/Typography';
+import { styled } from '@mui/material/styles';
+
+
+const DemoPaper = styled(Paper)(({ theme }) => (
```
[the remark this answers is not in the document]
Comment (lizzietm) [reply conversationally/frontend/src/pages/landingPage/sections/Testimonials.jsx:9]: Going to merge for the MVP and then work on this in the background.

**Review by lizzietm — COMMENTED** [review]
Going to merge for the MVP and then work on this in the background.

**Pull request merged**